src/DataFixtures/ProductFixtures.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace App\DataFixtures;
+
+use Doctrine\Common\DataFixtures\DependentFixtureInterface;
+use Doctrine\Bundle\FixturesBundle\Fixture;
+use Doctrine\Persistence\ObjectManager;
+use App\Entity\Product;
+use Exception;
+
+/**
+ * Class ProductFixtures
+ *
+ * @package App\DataFixtures
+ */
+class ProductFixtures extends Fixture implements DependentFixtureInterface
+{
+    /**
+     * @inheritDoc
+     *
+     * @throws Exception
+     */
+    public function load(ObjectManager $manager)
+    {
+        for ($i = 1; $i <= 10; $i++){
+            $product = new Product();
+            $product->setName('phone n' . $i);
+            $product->setReference('ref' . $i);
+            $product->setDescription('phone 1' . $i);
+            $product->setBrand($this->getReference(BrandFixtures::BRAND_REFERENCE));
```

Comment:
Tu dois faire appel à une des reférences que tu auras créé `$product->setBrand($this->getReference(BrandFixtures::BRAND_REFERENCE . "_1"));` en essayer de plutôt créer une boucle imbriqué